extensions/amp-consent/0.1/consent-info.js

```diff
@@ -127,14 +127,6 @@ export function recalculateConsentStateValue(newState, previousState) {
  * @return {?boolean|Object}
  */
 export function composeStoreValue(consentInfo, opt_forceNew) {
-  if (
-    !opt_forceNew &&
-    !consentInfo['consentString'] &&
-    consentInfo['isDirty'] === undefined
-  ) {
-    // TODO: Remove after turn on amp-consent-v2
-    return calculateLegacyStateValue(consentInfo['consentState']);
-  }
```

Comment:
Does this make sense?

Reply:
I think the `opt_forceNew` is always set to true when `amp-consent-v2` is turned on. Please double check, if that's the case we should also remove the param from the function call.

Reply:
Yes that is true. Removed